src/v3/src/components/ImageWithText/ImageWithText.tsx

```diff
@@ -47,6 +48,7 @@ const ImageWithText: UISchemaElementComponent<{
         <Box>
           <Typography
             component="span"
+            className={noTranslate ? 'no-translate' : undefined}
```

Comment:
Why do we need a no-translate here? Is the content of the text supposed to represent an image so we don't want to count it as an "English Leak"? 

Although if that is the case, would we ever want "localized" images?

Reply:
This text is a number, it is the code that is pushed to their OV app. SIW v2 expects it to have the no-translate class on it so their english leaks test can ignore it. 

FYI, I did create the spike for our team dependent on the ticket you shared yesterday 🙏